src/renderer/navigation/index.ts

```diff
@@ -0,0 +1,8 @@
+// Navigation (renderer)
+
+import { bindEvents } from "./events";
+
+export * from "./history";
+export * from "./helpers";
+
+bindEvents();
```

Comment:
Do we have to run this automatically on import?

Reply:
No, it's called only once here.